Documentation/subcommands/api-service.md

```diff
@@ -0,0 +1,19 @@
+# rkt api-service
+
+## Overview
+
+The API service is designed to help users to get a well defined and structual result for listing and introspecting their pods and images.
+The API service is implemented with [gRPC](http://www.grpc.io/).
+The API service is designed not to be running as a root, thus it currently only provides read-only interfaces for now.
```

Comment:
Change to:
"The API service is designed to run without `root` privileges, and currently provides a read-only interface."